docs/source/notes/mps.rst

```diff
@@ -0,0 +1,13 @@
+.. _MPS-Backend:
```

Comment:
There aren't doc builds yet but I think we need to include this somewhere for it to be put on the website?

I would add the following at the end as a quick example:

```rst
To get started, simply move your Tensor and Module to the ``mps`` device:

.. code::

    mps_device = torch.device("mps")

    # Create a Tensor directly on the mps device
    x = torch.ones(5, device=mps_device)
    # Or
    x = torch.ones(5, device="mps")

    # Any operation happens on the GPU
    y = x * 2

    # Move your model to mps just like any other device
    model = YourFavoriteNet()
    model.to(mps_device)

    # Now every call runs on the GPU
    pred = model(x)

```

Reply:
Done.